clueless/models.py

```diff
@@ -33,8 +33,8 @@ class Space(models.Model):
     """
     posX = models.IntegerField()
     posY = models.IntegerField()
-    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True)
-    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True)
+    spaceNorth = models.OneToOneField('self', related_name='spaceSouth', blank=True, null=True)
+    spaceEast = models.OneToOneField('self', related_name='spaceWest', blank=True, null=True)
     spaceCollector = models.ForeignKey(SpaceCollection)
```

Comment:
I found out it was needed on Postgres for blank FK references (or at least it wasn't working when I tried).